yarl/_url.py

```diff
@@ -1204,29 +1204,29 @@ def _get_str_query(self, *args: Any, **kwargs: Any) -> Union[str, None]:
             raise ValueError("Either kwargs or single query parameter must be present")
 
         if query is None:
-            query = None
-        elif isinstance(query, Mapping):
+            return None
+        if isinstance(query, (MultiDict, MultiDictProxy)):
+            return self._get_str_query_from_iterable(query.items())
```

Comment:
```suggestion
```

Would be nice to do this, and its probably a bugfix, but its already a breaking change since even though the docs don't say we do, we allowed lists/tuples inside of multidict